### PR TITLE
Media services 2019 05 01 preview

### PR DESCRIPTION
**Pull request by BrianBlum**
@giakas, @sanbhatt This is the public preview of the Live Transcription feature.

### CODE REVIEW

**Comment by BrianBlum** [comment]
@erich-wang , can we please get a review of these changes? Thanks

File: sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ namespace Media.Tests.ScenarioTests
 {
     public class LiveEventTests : MediaScenarioTestBase
     {
-        [Fact]
+        [Fact(Skip = "Need to rerun for 2019-05-01-preview")]
```
Comment (isra-fel) [inline sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs]: Hi Brian, as public preview is still shipped to all the customers, we need to ensure its quality aka proper test coverage. Wouldn't want to skip them

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs (hunk 1 above)
Comment (BrianBlum) [reply sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs]: @isra-fel, the tests that are marked skip don't test the new feature. We want to skip them to save the feature team from having to rerecord the test with APIs that use the version 2019-05-01-preview and then rerecord them again when we move the feature into the stable version - 2018-07-01, regardless if they choose to update the test with new API calls.
The new feature that we're previewing in 2019-05-01-preview has been thoroughly tested with both the .NET SDK and the Java SDK.

**Comment by BrianBlum** [comment]
@isra-fel , are you going to merge this PR with the skipped tests?

**Comment by isra-fel** [comment]
Hi Brian, I understand that you want to save the effort of devs, it's just we don't want to release anything not tested, especially when its for public.

Besides tests, there's supposed to be a metadata file under `eng/mgmt/mgmtmetadata/`, which will be generated when running `generate.ps1`.

File: sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs (hunk 1 above)
Comment (BrianBlum) [reply sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs]: @isra-fel, I have removed the skip attribute and rerecorded the tests. Please review.
Thanks

**Review by BrianBlum — COMMENTED** [review]
@isra-fel , I have updated the tests per your request.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**